pkg/console/subresource/deployment/deployment.go

```diff
@@ -206,6 +225,7 @@ func consoleContainer(cr *operatorv1.Console) corev1.Container {
 			"--public-dir=/opt/bridge/static",
 			"--config=/var/console-config/console-config.yaml",
 			"--service-ca-file=/var/service-ca/service-ca.crt",
+			flag,
```

Comment:
Adding both, flag and argument(above), so the log-level is passed to the `bridge` binary